dist/aframe-altspace-component.js

```diff
@@ -120,6 +106,86 @@
 	   * Use to continue or add any dynamic or background behavior such as events.
 	   */
 	  play: function () { },
+
+
+	  /********** Helper Methods **********/
+
+	  /**
+	   * Swap in Altspace renderer when running in AltspaceVR.
+	   */
+	  initRenderer: function () {
+
+	    var scene = this.el.object3D;
+	    if (!this.data.usePixelScale) {
+	      console.log('useMeterScale');//XXX
+	      altspace.getEnclosure().then(function(e) {
+	        scene.scale.multiplyScalar(e.pixelsPerMeter);
+	      });
+	    }
+	    var renderer = this.el.renderer = altspace.getThreeJSRenderer();
+	    var noop = function() {};
+	    renderer.setSize = noop;
+	    renderer.setPixelRatio = noop;
+	    renderer.setClearColor = noop;
+	    renderer.clear = noop;
+	    renderer.enableScissorTest = noop;
+	    renderer.setScissor = noop;
+	    renderer.setViewport = noop;
+	    renderer.getPixelRatio = noop;
+	    renderer.getMaxAnisotropy = noop;
+	    renderer.setFaceCulling = noop;
+	    renderer.context = {canvas: {}};
+	    renderer.shadowMap = {};
+
+	  },  
+
+	  /**
+	   * Emulate A-Frame cursor events when running in altspaceVR.
+	   */
+	  initCursorEvents: function() {
+
+	    var scene = this.el.object3D;
+	    var cursorEl = document.querySelector('a-cursor') || document.querySelector('a-entity[cursor]');
+	    if (cursorEl) { 
+	      // Hide A-Frame cursor mesh.
+	      cursorEl.setAttribute('material', 'transparent', true);
+	      cursorEl.setAttribute('material', 'opacity', 0.0);
```

Comment:
Would be better to set `material.visible = false` if possible. Turning on transparency uses a more expensive shader.